### DISCUSSION TITLE
donnez moi vos noms utilisateur gitHub

### DISCUSSION THREAD

**Discussion by guillaumeagile** [guillaumeagile/LPRO3_DQL_2021#4] [Announcements]
pour que je vous ajoute dans la liste des collaborateurs du projet

**Comment by bas22tien** [comment]
username bas22tien

**Reply by Spontex-microfibre** [reply]
Spontex-microfibre

**Reply by KevinLrmr** [reply]
KevinLrmr  
(Kévin LOURMIERE)

**Reply by GauvainDe** [reply]
GauvainDe
Gauvain Delapierre

**Reply by Paul31000** [reply]
Paul31000

**Comment by DevLudo** [comment]
DevLudo (Ludovic ROUX) travaille avec Jérémie Blanc et Pierre Alborghetti

**Comment by Azernox** [comment]
Julien LAPEYRADE

**Comment by Fargrim74** [comment]
Fauzzi Joris

**Comment by Ademo93** [comment]
Rayan dib

**Comment by TBoltsp** [comment]
Sylvain PAULET

**Comment by ghost** [comment]
Robin Kaczmareck

**Reply by ghost** [reply]
+ Hadrien Cellier

**Comment by Thomas-Dej** [comment]
Thomas De Jesus

**Reply by SebWayz** [reply]
+ Sébastien Bougouin

**Comment by piscouman** [comment]
Eliott Bourgeret avec Romain Sales
